hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,228 @@
+                </Button>
+            </form>
+        </>
+    );
+};
+
+export const EnhancedCartForm = ({
+    handleSubmit,
+    requestFailure,
+    availableQuantity,
+}) => {
+    const onSubmit = (formikValues) => {
+        handleSubmit(formikValues.quantity);
+    };
+
+    return (
+        handleSubmit && (
+            <Formik
+                initialValues={{ quantity: "1" }}
+                onSubmit={onSubmit}
+                validateOnBlur={false}
+                validationOnChange={false}
+                validationSchema={false}
+            >
+                {(formikProps) => (
+                    <CartForm
+                        availableQuantity={availableQuantity}
+                        handleSubmit={formikProps.handleSubmit}
+                        handleChange={formikProps.handleChange}
+                        requestFailure={requestFailure}
+                        values={formikProps.values}
+                    />
+                )}
+            </Formik>
+        )
+    );
+};
+
+const DetailInfoSection = ({
+    manufacturer,
+    model_num,
+    datasheet,
+    notes,
+    constraints,
+}) => {
+    return (
+        <div className={styles.detailinfosection}>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Constraints
+                </Typography>
+                {constraints.map((constraint, i) => (
+                    <Typography key={i} variant="body2">
+                        {constraint}
+                    </Typography>
+                ))}
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Manufacturer
+                </Typography>
+                <Typography>{manufacturer}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Model Number
+                </Typography>
+                <Typography>{model_num}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Datasheet
+                </Typography>
+                <Link
+                    href={datasheet}
+                    rel="noopener"
+                    color="inherit"
+                    underline="none"
+                    target="_blank"
+                    className={styles.bodyinfoDataSheet}
+                >
+                    <LaunchIcon></LaunchIcon>
+                    <Typography>Link</Typography>
+                </Link>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Notes
+                </Typography>
+                {notes.map((note, i) => (
```

Comment:
Notes in the models is just a text field, not a list, so this can just spit the content onto the page like the rest of the fields https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/hardware/models.py#L28

Reply:
does this also apply to the constraints which is also use `.map`

Reply:
Nope - constraints are actually a list. They're derived from the max per team counts for the item and category... some massaging into a single array will be required when we're processing the API response.